src/utils/eventList.js

```diff
@@ -0,0 +1,14 @@
+const guildScheduledEventCreate = require('../events/guildScheduledEventCreate');
+const interactionCreateEvent = require('../events/interactionCreate');
+const pollVotationResultEvent = require('../events/pollVotationResults');
+const qaMetionEvent = require('../events/qaMention');
+const readyEvent = require('../events/ready');
+
+const eventList = [
+  guildScheduledEventCreate,
+  interactionCreateEvent,
+  pollVotationResultEvent,
+  qaMetionEvent,
+  readyEvent,
+];
```

Comment:
I think it is like hidden here in utils, maybe put it outside of the folder, since it is not an util